src/bpf_loader.rs

```diff
@@ -0,0 +1,19 @@
+//! BPF loader
+use native_loader;
+use solana_program_interface::account::Account;
+use solana_program_interface::pubkey::Pubkey;
+
+pub const BPF_LOADER_PROGRAM_ID: [u8; 32] = [6u8; 32];
+pub const BPF_LOADER_NAME: &str = "bpf_loader";
+
+pub fn id() -> Pubkey {
```

Comment:
nit: `program_id()` seems nicer.   the name `id()` is too generic.  But I see `native_loader::id()` so we're already confused.   I sorta just want to delete this function entirely, it barely adds value

Reply:
I agree, just following convention with the rest of the programs.  If we change it should change it consistently across all the programs.  Mind if I leave it consistent with other programs and we can change them all in another pr?